idl/github.com/uber/cadence/admin.thrift

```diff
@@ -27,9 +27,9 @@ include "shared.thrift"
 **/
 service AdminService {
   /**
-  * InquiryWorkflowExecution returns information about the internal states of workflow execution.
+  * InquireWorkflowExecution returns information about the internal states of workflow execution.
   **/
-  InquiryWorkflowExecutionResponse InquiryWorkflowExecution(1: shared.DescribeWorkflowExecutionRequest inquiryRequest)
+  InquireWorkflowExecutionResponse InquireWorkflowExecution(1: InquireWorkflowExecutionRequest inquireRequest)
```

Comment:
let us use simple words here, try describe or get.

Reply:
+1 for Describe

Reply:
OK, I will rename them to DescribeXX